src/templates/csr/events-multiple/index.svelte

```diff
@@ -53,20 +54,44 @@
 		},
 	];
 
+	events = events.filter((event) => event.eventTitle !== '');
+
 	let height: number = -1;
 </script>
 
 <aside bind:clientHeight={height}>
-	<EventsHeader {HeaderButtonUrl} {BannerDescription} {HeaderButtonText} />
+	<ManualHeader
+		buttonText={HeaderButtonText}
+		buttonUrl={HeaderButtonUrl}
+		backgroundColour="#c83877"
+		buttonColour="#ffffff"
+	>
+		<svelte:fragment slot="logo">
+			<LiveLogo />
+		</svelte:fragment>
+		<svelte:fragment slot="description">
+			Our events are a unique opportunity to hear Guardian journalists discuss
```

Comment:
The description should be set to the BannerDescription variable as it's something that we manually choose

Reply:
I'm not sure where this text came from 😅